IncludeToolbox/Commands/FormatIncludes.cs

```diff
@@ -79,7 +79,7 @@ protected override void MenuItemCallback(object sender, EventArgs e)
                     line.UpdateTextWithIncludeContent();
 
                 // Sorting. Ignores non-include lines.
-                IncludeFormatter.IncludeFormatter.SortIncludes(lines, settings.SortByType, settings.RegexIncludeDelimiter, settings.PrecedenceRegexes, document.Name);
+                IncludeFormatter.IncludeFormatter.SortIncludes(lines, settings, document.Name);
```

Comment:
SortIncludes started to require too many settings so just pass in the whole settings object.